File: BAEKJOON/Python/2000/2400/2480.py
"""
Title : 주사위 세개
Link : https://www.acmicpc.net/problem/2480
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    X, Y, Z = map(int, input().split())
    if X == Y == Z:
        print(10000 + X * 1000)
    elif X == Y:
        print(1000 + X * 100)
    elif Y == Z:
        print(1000 + Y * 100)
    elif Z == X:
        print(1000 + Z * 100)
    else:
        print(max(X, Y, Z) * 100)

File: BAEKJOON/Javascript/2000/2400/2480.js
const fs = require("fs")
const filepath = process.platform === "linux" ? "/dev/stdin" : "./input.txt"
const input = fs.readFileSync(filepath).toString().trim()

const [X, Y, Z] = input.split(" ").map(x => parseInt(x))
if (X === Y && Y === Z) {
  console.log(10000 + 1000 * X)
} else if (X === Y) {
  console.log(1000 + 100 * X)
} else if (Y === Z) {
  console.log(1000 + 100 * Y)
} else if (Z === X) {
  console.log(1000 + 100 * Z)
} else {
  maxValue = Math.max(X, Y, Z)
  console.log(maxValue * 100)
}
